Mathlib/Data/ByteArray.lean

```diff
@@ -27,6 +32,7 @@ def upRel (ub : Nat) : WellFoundedRelation Nat := ⟨Up ub, Up.WF ub⟩
 end Nat
```

Comment:
Do we not want to deprecated `upRel` as well?

Reply:
I decided it wasn't necessary it individually deprecate things, once the types they talked about were deprecated... This is all unused code.

Reply:
This definition doesn't talk about any deprecated types though